dialogue-clients/src/test/java/com/palantir/dialogue/clients/ReloadingClientFactoryTest.java

```diff
@@ -91,4 +99,42 @@ void live_reloading_wrapper_still_uses_the_EndpointChannelFactory_factory() {
         // ensure we use the bind method
         verify(channel, atLeastOnce()).endpoint(any());
     }
+
+    @Test
+    void withConjureErrorParameterSerializationFormat_wraps_runtime_with_specified_format() {
+        ChannelCache mockCache = mock(ChannelCache.class);
+        ImmutableReloadingParams params = ImmutableReloadingParams.builder()
+                .scb(Refreshable.only(ServicesConfigBlock.builder().build()))
+                .build();
+        ReloadingClientFactory factory = new ReloadingClientFactory(params, mockCache);
+
+        // Get the original runtime
+        ConjureRuntime originalRuntime = params.runtime();
+        assertThat(originalRuntime.bodySerDe().errorParameterFormat()).isEqualTo(Optional.empty());
```

Comment:
nit: I think you can simply do
```suggestion
        assertThat(originalRuntime.bodySerDe().errorParameterFormat()).isEmpty();
```